components/script/dom/element.rs

```diff
@@ -150,6 +150,7 @@ pub struct Element {
     /// https://dom.spec.whatwg.org/#concept-element-custom-element-definition
     #[ignore_heap_size_of = "Rc"]
     custom_element_definition: DOMRefCell<Option<Rc<CustomElementDefinition>>>,
+    custom_element_state: Cell<CustomElementState>,
```

Comment:
Add a link to https://dom.spec.whatwg.org/#concept-element-custom-element-state .